Model/Calendar.php

```diff
@@ -14,12 +14,14 @@ class Calendar
      */
     private $tz;
 
-    public function __construct(Timezone $tz)
+    public function __construct(Timezone $tz = null)
```

Comment:
Can you add a phpdoc to explain why Timezone can be null ("for compatibility with Outlook, we need...") ?